src/test/store/actions.test.js

```diff
@@ -384,3 +387,111 @@ describe('actions/updateProfileSelection', function() {
     });
   });
 });
+
+describe('app actions', function() {
```

Comment:
all good, but can you please move this to a separate file app.test.js ?

Reply:
Done.